polaris/polaris/utils.py

```diff
@@ -443,6 +436,16 @@ def create_transaction_envelope(transaction, source_account) -> TransactionEnvel
     return builder.build()
 
 
+def get_balanceid(response):
```

Comment:
Nice. 

can we change the name to to match the attribute? `get_balance_id()`? 

I would also add a docstring explaining the return value. Why hex? Is that according to the CAP/SEP? Or would base64 make more sense? Keep in mind Horizon uses base64 for encoding bytes across the board in their API's. The SDK's on the other hand largely expect hex. I would stick to this pattern: base64 for bytes sent or received in requests, hex if will only be used internally. 

Another good way to think about it is, will developers call `get_balance_id()` and immediately call `base64.b64encode()`?

Reply:
This in order to query and claim claimable balances (on horizon) you need the hex representation of said id.
So to reduce friction I convert this id to hex.

Reply:
Oh I see, so _Horizon uses base64 for encoding bytes across the board in their API's._ isn't true. sg